databases/week2/db_2of_week2.sql

```diff
@@ -0,0 +1,28 @@
+USE hyf_lesson2;
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT *
+FROM task
+JOIN user_task on task.id=user_task.task_id
+JOIN user ON user.id=user_task.user_id
+WHERE user.email LIKE '%@spotify.com';
+--Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT *
+FROM task
+JOIN user_task on task.id=user_task.task_id
+JOIN user ON user.id=user_task.user_id
+WHERE user.name='Donald Duck' AND ;
+
+--Get all the tasks for 'Maryrose Meadows' that were created in september (hint: month(created)=month_number)
+SELECT *
+FROM task
+JOIN user_task on task.id=user_task.task_id
+JOIN user ON user.id=user_task.user_id
+WHERE user.name='Maryrose Meadows' AND created LIKE '%09%';
```

Comment:
it is not correct, how do you guarantee it will look for 09 on Month ? maybe 1090 in year ? or 2009? or 09 in day ?